web/themes/joinup/templates/content/node--document--listing-item.html.twig

```diff
@@ -86,9 +86,7 @@
       {{ title_prefix }}
       <h2 class="listing__title"><a href="{{ url }}">{{ label }}</a></h2>
       {{ title_suffix }}
-      {% if content.field_type|render %}
```

Comment:
Honestly, I don't quite understand why in some places the check is added, such as in `{% if content.field_keywords|render %}` and in others it's removed.

Reply:
Probably Tom added these checks only for fields which are not mandatory to avoid printing empty markup?